### DISCUSSION TITLE
keycloak-js-bower removed

### DISCUSSION THREAD

**Discussion by SeRodrigalvarez** [keycloak/keycloak#9651] [Q&A] [answered]
I'm integrating a legacy app with Keycloak. Frontend part of this app uses bower to manage dependencies. I added keycloak-js-bower dependency and it worked just fine some month ago, but today, while building a docker image of this app, I got an error fetching this dependency:

bower keycloak#^10.0.2                                      ECMDERR Failed to execute "git ls-remote --tags --heads https://github.com/keycloak/keycloak-js-bower.git", exit code of #128 Logon failed, use ctrl+c to cancel basic credential prompt. remote: Invalid username or password. fatal: Authentication failed for 'https://github.com/keycloak/keycloak-js-bower.git/'

It turns out that https://github.com/keycloak/keycloak-js-bower.git was removed for some reason and bower is trying to fetch the dependecy from there. Is there any reason why it was removed? Any ideas of how to address this issue?

**Comment by stianst** [comment, marked as answer]
Probably just as well we removed it as we haven't published any updates there for a long time. I'd suggest changing to pulling the adapter from NPM instead.
